docs/content/en/docs/getting-started/production-environment/gs-mgmt-cluster.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Create production cluster"
+weight: 40
+---
+
+EKS Anywhere supports a vSphere provider for production grade EKS Anywhere deployments.
+EKS Anywhere allows you to provision and manage Amazon EKS on your own infrastructure.
+
+This document walks you through setting up EKS Anywhere in a way that:
+
+* Deploys a management cluster on your vSphere environment
+* Deploys one or more workload clusters from the management cluster
+* Keeps the management cluster in place so you can use it to later upgrade and delete workload clusters
+
+% alert title="Important" color="primary" %
+
+In the initial release, EKS Anywhere clusters functioned as both workload and management clusters.
+CAPI functions moved back and forth before between temporary bootstrap clusters running on your administrative machine and the combined cluster.
+
+Separating management features into a separate, persistant management cluster
+provides a cleaner model for managing the lifecycle of workload clusters (to create, upgrade, and delete clusters), while workload clusters run user applications.
+This approach also allows for segmentation of different cluster types.
```

Comment:
You may want to add " and reduces provider permissions for workload clusters."

Reply:
Good point. Done.